bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/handler/IntesisBoxHandler.java

```diff
@@ -0,0 +1,383 @@
+                logger.trace("Sending command {} to function {}", value, function);
+                api.sendCommand(function, value);
+            } else {
+                logger.warn("Sending command failed, could not get API");
+            }
+        }
+    }
+
+    private void receivedId(String function, String value) {
+        logger.trace("receivedID(): {} {}", function, value);
+        properties.put(PROPERTY_VENDOR, "Intesis");
+        switch (function) {
+            case "MODEL":
+                properties.put(PROPERTY_MODEL_ID, value);
+                break;
+            case "MAC":
+                properties.put(PROPERTY_MAC_ADDRESS, value);
+                break;
+            case "IP":
+                properties.put("ipAddress", value);
+            case "PROTOCOL":
+                properties.put("protocol", value);
+                break;
+            case "VERSION":
+                properties.put(PROPERTY_FIRMWARE_VERSION, value);
+                break;
+            case "RSSI":
+                properties.put("rssi", value);
+                break;
+            case "NAME":
+                properties.put("hostname", value);
+                break;
+        }
+        logger.trace("Update Properties for ID : {}", properties);
+    }
+
+    private void receivedUpdate(String function, String receivedValue) {
+        String value = receivedValue;
+        logger.trace("receivedUpdate(): {} {}", function, value);
+        switch (function) {
+            case "ONOFF":
+                updateState(CHANNEL_TYPE_POWER, OnOffType.from(value));
+                break;
+
+            case "SETPTEMP":
+                if (value.equals("32768")) {
+                    value = "0";
+                }
+                updateState(CHANNEL_TYPE_TARGETTEMP,
+                        new QuantityType<Temperature>(Double.valueOf(value) / 10.0d, SIUnits.CELSIUS));
+                break;
+            case "AMBTEMP":
+                if (Double.valueOf(value).isNaN()) {
+                    value = "0";
+                }
+                updateState(CHANNEL_TYPE_AMBIENTTEMP,
+                        new QuantityType<Temperature>(Double.valueOf(value) / 10.0d, SIUnits.CELSIUS));
+                break;
+            case "MODE":
+                updateState(CHANNEL_TYPE_MODE, new StringType(value));
+                break;
+            case "FANSP":
+                updateState(CHANNEL_TYPE_FANSPEED, new StringType(value));
+                break;
+            case "VANEUD":
+                updateState(CHANNEL_TYPE_VANESUD, new StringType(value));
+                break;
+            case "VANELR":
+                updateState(CHANNEL_TYPE_VANESLR, new StringType(value));
+                break;
+            case "ERRCODE":
+                properties.put("errorCode", value);
+                updateProperties(properties);
+                break;
+            case "ERRSTATUS":
+                properties.put("errorStatus", value);
```

Comment:
Why are errorCode and errorStatus properties instead of channels?

Reply:
Changed to be channels